Add --silent-output runtime option

alessandropellegrini opened the pull request:
When running a model specifying `--silent-output`, any output generated
by both the model and the runtime is suppressed.

This is useful in production, when one is interested only in simulation
results (e.g. performance data in the output folder) or in simulation
results related to the committed portion, which should be saved to file
in `OnGVT()`.

This closes #2.

Signed-off-by: Alessandro Pellegrini <pellegrini@dis.uniroma1.it>

alessandropellegrini committed:
Add --silent-output runtime option

When running a model specifying `--silent-output`, any output generated
by both the model and the runtime is suppressed.

This is useful in production, when one is interested only in simulation
results (e.g. performance data in the output folder) or in simulation
results related to the committed portion, which should be saved to file
in `OnGVT()`.

This closes #2.

Signed-off-by: Alessandro Pellegrini <pellegrini@dis.uniroma1.it>

Piccions committed:
Removed an empty statement

Piccions committed:
Correction of the silent output test

Piccions reviewed (dismissed):
The continuous integration complains because of a bug in the PCS model.
This PR is accepted anyway (let's clean these models!).

Piccions committed:
Hard fix on the PCS model

The PCS model in this branch has some ugly leftovers. I removed the minimum needed to let the CI pass.

Piccions commented on models/pcs/application.c:
```diff
@@ -247,10 +247,6 @@ void ProcessEvent(unsigned int me, simtime_t now, int event_type, event_content_
 			state->arriving_handoffs++;
 			state->arriving_calls++;
 
-			if(Random() < 0.3 && me == 1 && event_content->from == 2){//&& state->dummy_flag == false) {
-				*(event_content->dummy) = 1;
-				state->dummy_flag = true;
-			}
```
!!!

Piccions reviewed (approved):
Approved!
